lib/chainsync/chainsync/db/hyperdrive/schema.py

```diff
@@ -10,6 +10,11 @@
 
 # pylint: disable=invalid-name
 
+# Postgres numeric type that matches fixedpoint
+# The high precision doesn't actually allocate memory in postgres, as numeric is variable size
+# https://stackoverflow.com/questions/40686571/performance-of-numeric-type-with-high-precisions-and-scales-in-postgresql
+FIXED_NUMERIC = Numeric(precision=1000, scale=18)
```

Comment:
Can you add a note to this comment saying "Precision indicates the total number of digits, while scale indicates the the number of digits to the right of the decimal."

I ask because according to the [Python Decimal type](https://docs.python.org/3/library/decimal.html#:~:text=Unlike%20hardware%20based%20binary%20floating%20point%2C%20the%20decimal%20module%20has%20a%20user%20alterable%20precision%20(defaulting%20to%2028%20places)%20which%20can%20be%20as%20large%20as%20needed%20for%20a%20given%20problem%3A), the "precision" specifies how many decimal places we support (i.e. to the right of the decimal point). While sqlalchemy uses the opposite (not well defined in their docs, but I think it is outlined [here](https://www.w3schools.com/sql/sql_datatypes.asp#:~:text=decimal(p%2Cs,and%20scale%20numbers)).